Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -593,6 +595,8 @@ open class MapView: UIView {
             return
         }
 
+        subscribeToLifecycleNotifications()
+
```

Comment:
Now that we're using these lifecycle notifications in a more general-purpose way, would it be simpler to just subscribe at the end of commonInit and stay subscribed all the time?

Reply:
Being subscribed all the time doesn't seem like it would be significant in terms of performance since the handlers do very little work and are called infrequently. Reducing memory use would be valuable regardless of whether the MapView belongs to a window. Besides performance, are there any other things we should be concerned about?

Reply:
good point, I will remove the unsubscribe line and make MapView subscribe to these notifications always